service/src/main/resources/api/service_openapi.yaml

```diff
@@ -353,12 +350,9 @@ components:
           type: string
           description: Unique identifier of the Azure subscription
           format: uuid
-        resourceGroupName:
-          type: string
-          description: Name of the Azure resource group
-        applicationDeploymentName:
+        managedResourceGroupId:
           type: string
-          description: Name of the Azure application deployment
+          description: Managed resource group ID of the Terra deployment
```

Comment:
Nit - maybe "Terra managed application deployment" (I read it Terra control plane deployment in my head)